### ISSUE TITLE
Selectively hide contacts

### ISSUE THREAD

**Issue by exquo** [isamert/scli#228]
### Discussed in https://github.com/isamert/scli/discussions/226

<div type='discussions-op-text'>

Add an option to not show (or show separately) some of the contacts.

**Comment by lukebarone** [comment]
Agreed - my Contacts list shows EVERYONE who I SMS'd with Signal back when it was supported. Would love if it only showed current Signal contacts.
